build_tools/docker/manage_images.py

```diff
@@ -18,7 +18,7 @@
   python3 build_tools/docker/manage_images.py --image cmake
 
 Print out output for rebuilding the cmake image and all images that
-transitively on depend on it, but don't take side-effecting actions:
+transitively depend on it, but don"t take side-effecting actions:
```

Comment:
fixed one grammar issue but broke some punctuation with an over-zealous find-replace on `'` to `"` :P

```suggestion
transitively depend on it, but don't take side-effecting actions:
```